### ISSUE TITLE
Hiding sklearnex verbose logging on Kaggle

### ISSUE THREAD

**Issue by Innixma** [autogluon/autogluon#4141]
We should look into hiding the sklearnex verbose logs on Kaggle. For some reason on Kaggle these logs are enabled by default (but they aren't enabled in other environments outside of Kaggle).

Example: https://www.kaggle.com/code/prishasawhney/obesity-autogluon-log-loss#Step-3:-Initiate-Autogluon

Solution from maintainers are to either avoid editing the root logger verbosity, or otherwise do the following(@napetrov):

```
import logging
logging.getLogger('sklearnex').setLevel("WARNING")
```

Related: https://github.com/intel/scikit-learn-intelex/issues/1695

Example of Logs on Kaggle: https://www.kaggle.com/competitions/playground-series-s4e5/discussion/499495#2789785
